junitparser/junitparser.py

```diff
@@ -360,22 +365,25 @@ def is_skipped(self):
         return any(isinstance(r, Skipped) for r in self.result)
 
     @property
-    def result(self):
+    def result(self) -> List[FinalResult]:
         """A list of :class:`Failure`, :class:`Skipped`, or :class:`Error` objects."""
-        return [r for r in self if isinstance(r, tuple(POSSIBLE_RESULTS))]
+        return [r for r in self if isinstance(r, FinalResult)]
```

Comment:
does this work? can `isinstance` implicitly convert the set into a tuple?

[docs say](https://docs.python.org/3/library/functions.html#isinstance): 
> [ ... ] classinfo is a tuple of type objects (or recursively, other such tuples) or a [Union Type](https://docs.python.org/3/library/stdtypes.html#types-union) of multiple types
> [ ... ]
> If classinfo is not a type or tuple of types and such tuples, a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) exception is raised

Reply:
I can't read, ignore this